DataFormats/L1Trigger/src/classes_def.xml

```diff
@@ -304,8 +304,8 @@
   <class name="edm::Wrapper<l1t::VertexWordCollection>"/>
   <class name="edm::Ref<l1t::VertexWordCollection>" />
 
-  <class name="l1t::TkJetWord" ClassVersion="3">
-    <version ClassVersion="3" checksum="3521396532"/>
+  <class name="l1t::TkJetWord" ClassVersion="4">
+    <version ClassVersion="4" checksum="2346039066"/>
```

Comment:
Can you keep the ClassVersion 3 below?

Reply:
@cecilecaillol How I can do that? This number is generated automatically (as far as I know). Is there a way to keep it bellow 3? If so I would be glad to do it.

Reply:
You should use ClassVersion 4 as automatically generated, but ClassVersion 3 details are still written below to keep track:
<class name="l1t::TkJetWord" ClassVersion="4">
    <version ClassVersion="4" checksum="2346039066"/>
    <version ClassVersion="3" checksum="3521396532"/>

Reply:
Hi Cecile,

Ok If I understand correctly, use the class v4 but *keep* the v3 lines as well (like comment) right?

Reply:
yes, see eg https://github.com/cms-sw/cmssw/blob/a47330117f767cde1895e61e155f9bd14dbbad22/DataFormats/L1Trigger/src/classes_def.xml#L2-L5

Reply:
Thanks a lot. I will do it today and update the PR!

Reply:
Dear Cecile,

I did it in my new update. Let me know if you have more questions